pkg/apis/app/v2/kieapp_types.go

```diff
@@ -942,6 +944,26 @@ type ObjectReference struct {
 	FieldPath string `json:"fieldPath,omitempty" protobuf:"bytes,7,opt,name=fieldPath"`
 }
 
+// KafkaExt kafka configuration to be used by the KieApp
+type KafkaExt struct {
+	// Disable the Kafka Extension
+	ExtDisabled string `json:"extDisabled,omitempty"`
+	// A comma separated list of pairs <message/signal name>=<topicName>
+	Topics string `json:"topics,omitempty"`
+	// A comma separated list of bootstrapServers
+	BootstrapServers string `json:"bootstrapServers,omitempty"`
+	// Identifier to pass to the server when making requests
+	ClientID string `json:"clientID,omitempty"`
+	// Allow automatic topic creation.
+	AutocreateTopics string `json:"autocreateTopics,omitempty"`
+	// Group identifier the group this consumer belongs
+	GroupID string `json:"groupID,omitempty"`
+	// The number of acknowledgments the producer requires the leader to have received before considering a request complete.
+	Acks string `json:"acks,omitempty"`
+	// Number of milliseconds that indicates how long publish method will block
+	MaxBlockMs string `json:"maxBlockMs,omitempty"`
```

Comment:
int64

Reply:
int is better to start, are milliseconds to express seconds to block